examples/build_file_generation/WORKSPACE

```diff
@@ -28,13 +28,16 @@ http_archive(
 )
 
 # Download the bazel_gazelle ruleset.
+GAZELLE_VERSION = "0.33.0"
```

Comment:
This is actually a bad refactor, especially to put into examples. Many people use renovate to bump dependencies and it won't understand this. Could you bump the dependency without making this refactor, please?

Reply:
Ah I see, I hadn't realised that was why it was setup that way. I will make that change :)

Reply:
done -- will rerequest review when CI passes